client/components/LinkGeo/linkGeo.scss

```diff
@@ -0,0 +1 @@
+@import 'client/style/partials';
```

Comment:
Add new line at EOF :)

Reply:
@sorja @xbexbex shouldn't/didn't the pre-commit hook take care of that ?

Reply:
I guess it didn't. Fixed it manually now.